<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QPushButton" name="downloadLocationBtn">
             <property name="text">
              <string>Select Download Location</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Public Name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Speed Limit</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Exchange URL</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="4">
            <widget class="QLineEdit" name="exchangeURL"/>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="publicName"/>
           </item>
           <item row="0" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QDoubleSpinBox" name="speedLimitSpin">
             <property name="suffix">
              <string> Mbps</string>
             </property>
             <property name="minimum">
              <double>2.500000000000000</double>
             </property>
             <property name="maximum">
              <double>9.900000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
             <property name="value">
              <double>3.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QSpinBox" name="port">
             <property name="minimum">
              <number>1025</number>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>2121</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="3">
            <widget class="QSlider" name="speedLimitSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="4">
            <widget class="QLineEdit" name="donwloadLocationInput"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="sharedLocationBtn">
             <property name="text">
              <string>Select Shared Location</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="4">
            <widget class="QLineEdit" name="sharedLocationInput"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Don't randomly hit your mouse. It hurts!</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="toggleShareBtn">
             <property name="text">
              <string>Toggle Share</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="usersListTab">
      <attribute name="title">
       <string>Current Users</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="browserTab">
      <attribute name="title">
       <string>Browser</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="downlaodsTab">
      <attribute name="title">
       <string>Downloads</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
